### ISSUE TITLE
projects/core/howto/defer-intro.rst should be properly formatted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6948]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#6948|
|Type|task|
|Created|2014-02-06 12:09:30Z|
|Branch|https://github.com/twisted/twisted/tree/defer-intro-docs-format-6948|

As with [#6943](https://github.com/twisted/twisted/issues/6943), this is updating the rST sources to be correctly formatted. 

<details><summary>Searchable metadata</summary>

```
trac-id__6948 6948
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_defer_intro_docs_format_6948 branches/defer-intro-docs-format-6948
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1391688570000000 1391688570000000
changetime__1393054018000000 1393054018000000
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [41571]) Branching to defer-intro-docs-format-6948.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Putting this up for review.

Mostly formatting changes, switched some things to use oxford commas, and the like.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

I have checked the changes and they look good.

Maybe the variables names could be improved to a better coding standrad... x, f, g, h might be great for mathematicians, but in real code I always find them hard to read.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj set owner to @hawkowl|
|-|-|

Thanks hawkowl,

Yeah I think this looks ok too. It renders ok with "make html" all the links
seem to work. I skimmed through the whole rendered document without finding any obvious mistakes.

I did find it a bit "wordy". Not sure it's an easy read for non-native English speakers. But that's a separate issue.

The only thing I thing you should consider is that "Deferred <Deferred>`\s," looks a bit odd when rendered and looks annoying to type.

Personally I'd just say <Deferreds>. But up to you.

Please merge after answering or addressing the Deferreds syntax point and after answering or addressing adirobans points.

-RichardW.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

Hi rwall and adiroiban,

Thanks for the review.

This ticket is formatting, so I'd like to leave any changes like the variable letters to another one. They're also meant to be generic and meaningless, to remove any implication of context.

The <Deferred>\ s/Deferreds thing was from either glyph or exarkun in IRC, because it's a Deferred, not a Deferreds.

Will merge.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl set status to `closed`|
|-|-|

(In [41739]) Merge defer-intro-docs-format-6948: projects/core/howto/defer-intro.rst should be properly formatted

Author: hawkowl
Reviewers: adiroiban, rwall
Fixes: [#6948](https://github.com/twisted/twisted/issues/6948)
